woocommerce/wc-admin
# Add disabled prop to SelectControl

Fixes #6764

This PR adds the prop "disabled" to the SelectControl.

### Accessibility




### Screenshots
![screenshot-one wordpress test-2021 04 28-13_49_45](https://user-images.githubusercontent.com/1314156/116442321-bdb75900-a828-11eb-8993-9e9f85168a15.png)


### Detailed test instructions:

_To make this PR easier to test I created the branch  `add/6764_test`_
-   Checkout the branch `add/6764_test`.
-   Go to the first step of the OBW (`/wp-admin/admin.php?page=wc-admin&path=%2Fsetup-wizard&step=store-details`).
-   Verify the `Country / Region` selector is disabled.
-   Checkout the branch `add/6764` or remove the `isDisabled={ true }` from [here](https://github.com/woocommerce/woocommerce-admin/blob/add/6764_test/client/dashboard/components/settings/general/store-address.js#L209) if you want to stay on `add/6764_test`.
-   The `Country / Region` selector now should be enabled again.
